### PR TITLE
Ui motor control

### PR DESCRIPTION
**Pull request by nsmohan**
-Removed C# UI
- Created UI in tkinter (Able to move camera and drive motors) 
-Able to change RSXA settings

-Update driver code to allow array of messages. 

Closing #40

### CODE REVIEW

File: rui/lib/gui_application.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+        """ 
+        "  @brief Read the RSXA settings file
+        """
+
+        with open(LOCAL_RSXA_FILE, "r") as rsxa_file:
+            return json.loads(rsxa_file.read())
+
+
+    def __wrtie_rsxa_settings(self, rsxa_settings):
```
Comment (ChemistryMickey) [inline rui/lib/gui_application.py:49]: "write"

Surprised this didn't throw errors down the line

File: rui/lib/gui_application.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+              communicate with the nibot
+"  @author    Nitin Mohan
+"  @date      May 07, 2020
+"  @copyright 2020 - NM Technologies
+"""
+
+#---------------------------------------------------#
+#                   System Imports                  #
+#---------------------------------------------------#
+import zope.event
```
Comment (ChemistryMickey) [inline rui/lib/gui_application.py:16]: Sorry, after a quick google I can see that Zope is a web application framework, but I only see your 
`zope.event.notify("connected")`
call.

I also later see your imports of tkinter. What's the utility of having both TK (GUI interface) and Zope (web interface)?
(mostly for my learning purposes, not accusing)

File: rui/ui_comps/handle_motor_rq.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        # -- Check if Speed is provided --#
+        if speed != "" and motor in DRIVE_MOTORS:
+            try:
+                speed = int(speed)
+            except ValueError:
+                self.throw_error("Invalid Speed Provided!")
+        else:
+            speed = -1
+
+        #-- Check for Dual Dirve Motor Movement --#
```
Comment (ChemistryMickey) [inline rui/ui_comps/handle_motor_rq.py:66]: "Dual Drive"

File: rui/main.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        self.logo.image = render
+
+    def _layout(self):
+        self.logo.place(x=0, y=0)
+
+
+class LayoutBody(LayoutBase):
+
+    def _class_comps_init(self):
+        self.cam_ctrl = CameraControlBox(self.control_tab, self.nibot_ap)
```
Comment (nsmohan) [inline rui/main.py:60]: Remove objects that are not used.

File: rui/lib/gui_application.py (hunk 1 above)
Comment (nsmohan) [reply rui/lib/gui_application.py:49]: Fixed. Thanks!

File: rui/lib/gui_application.py (hunk 2 above)
Comment (nsmohan) [reply rui/lib/gui_application.py:16]: zope.event is just a library to raise/handle events across the application. Very light weight. You can easily setup subscribers and raise events "notify' this is primarily why I used it.

File: rui/ui_comps/handle_motor_rq.py (hunk 3 above)
Comment (nsmohan) [reply rui/ui_comps/handle_motor_rq.py:66]: Fixed .

**Pull request merged**